texar/modules/encoders/rnn_encoders.py

```diff
@@ -788,3 +804,23 @@ def output_layer_bw(self) -> Optional[nn.Module]:
         r"""The output layer of the backward RNN.
         """
         return self._output_layer_bw
+
+    @property
+    def output_size(self) -> Tuple[int, int]:
+        r"""Return the finale dimensions of :meth:`forward` output tensor.
```

Comment:
Could we unify the docstrings for all modules? As long as the base method `ModuleBase.output_size` has docstrings, I think it's okay to omit docstrings for certain modules if they don't require extra explanantions.